### DISCUSSION TITLE
GCP Cloud Storage Bucket with Laravel Spatie

### DISCUSSION THREAD

**Discussion by ABC2015** [austintoddj/canvas#1373] [Troubleshooting]
Hi Community:

Wanted to confirm that use the GCP storage bucket using Laravel Spatie for GCP cloud storage works as expected.  Having some trouble getting images to POST based on the network request we are seeing.

//Config below** 
    'storage_disk' => env('CANVAS_STORAGE_DISK', 'gcs'),

    'storage_path' => env('CANVAS_STORAGE_PATH', 'public/canvas'),

    'upload_filesize' => env('CANVAS_UPLOAD_FILESIZE', 1048576),

 **//General Storage Bucket**
        'gcs' => [
            'driver' => 'gcs',
            'key_file_path' => env('GOOGLE_CLOUD_KEY_FILE', null), // optional: /path/to/service-account.json
            'project_id' => env('GOOGLE_CLOUD_PROJECT_ID', 'your-project-id'), // optional: is included in key file
            'bucket' => env('GOOGLE_CLOUD_STORAGE_BUCKET', 'your-bucket'),
            'path_prefix' => env('GOOGLE_CLOUD_STORAGE_PATH_PREFIX', ""), // optional: /default/path/to/apply/in/bucket
            'storage_api_uri' => env('GOOGLE_CLOUD_STORAGE_API_URI', null), // see: Public URLs below
            'visibility' => 'private', // optional: public|private
            'metadata' => ['cacheControl' => 'public,max-age=86400'], // optional: default metadata
        ],

ISSUE:
Essentially the application post request to google cloud apis does not seem to be placing the file in the right path.
![upload image](https://github.com/austintoddj/canvas/assets/6826984/d48e89ae-9ec2-4f7a-aeeb-092ef32fa894)
![upload image2](https://github.com/austintoddj/canvas/assets/6826984/69088444-633f-4f99-9501-070952d70e30)

Video of Issue
https://app.screencast.com/dFyxElAfDPBvB
https://app.screencast.com/a1GGu1I3FpL2j

Any suggestions would be helpful.

**Comment by austintoddj** [comment]
Closing as obsolete with [Canvas v7.0.0](https://github.com/austintoddj/canvas/releases/tag/v7.0.0).
